app/routes/overview/index.tsx

```diff
@@ -252,6 +277,7 @@ export default function Index() {
               rel="noopener"
             >
               <LoadingButton
+                id="taskUseCaseLibButton"
```

Comment:
for html attribute value i'm used to use kebab-case, it's more easy to read inside the dom and for units / e2e testing. What do you think ?

Reply:
No problem ill fix that !